recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 if [ "$(uname)" == "Darwin" ]; then
     export CXX="${CXX} -stdlib=libc++"
     export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+    # Shared Fortran libs on OSX unsupported as of 1.8.18
+    FORTRAN_FLAGS=("--enable-fortran" "--enable-fortran2003")
```

Comment:
No we use a `gcc` package to handle that case. Considering this built fine before the `--enable-unsupported` flag was added and it is complaining about shared libraries on Mac, I think it would be worth trying to see if forcing static libraries would work. Would recommend borrowing this [commit]( https://github.com/conda-forge/python-feedstock/commit/f2537c24f16c9f92d10b7584e801f2293acdc74e ) and changing `libbz2` to `libgfortran`.